aep/general/0157/aep.md.j2

```diff
@@ -7,18 +7,12 @@ needs to give the user control over which fields it sends back.
 
 APIs **may** support partial responses in one of two ways:
 
-### Field masks parameter
+### `read_mask` parameter
 
-Field masks (`google.protobuf.FieldMask`) can be used for granting the user
-fine-grained control over what fields are returned. An API **should** support
-the mask in a side channel. For example, the parameter can be specified either
-using an HTTP query parameter, an HTTP header, or a [gRPC metadata entry][].
+Field masks can be used for granting the user fine-grained control over what
+fields are returned. An API **should** support the mask as a request field
+named `read_mask`.
 
-Field masks **should not** be specified in the
-[request](./0157.md#read-masks-as-a-request-field).
-
-- The value of the field mask parameter **must** be a
-  `google.protobuf.FieldMask`.
 - The field mask parameter **must** be optional:
   - An explicit value of `"*"` **should** be supported, and **must** return all
```

Comment:
Does this match up with what we've decided for reading across collections? 

(That's a URI parameter and this is a field value, but we should be explicit about when/how we use `-` vs `*`)

Reply:
Yeah I see your point - but I think a `*` in the path is different from one in a request field, and I also don't think we can change the FieldMask spec in this way. :/